packages/zosjobs/src/SubmitJobs.ts

```diff
@@ -53,21 +55,43 @@ export class SubmitJobs {
     }
 
     /**
-     * Submit a job that resides in a z/OS data set.
+     * Submit a job that resides in a USS File.
+     * @static
+     * @param {AbstractSession} session - z/OSMF connection info
+     * @param {string} jobUSSFile - job USS File to be translated into parms object
+     * @returns {Promise<IJob>} - Promise that resolves to an IJob document with details about the submitted job
+     * @memberof SubmitJobs
+     */
+    public static submitUSSJob(session: AbstractSession, jobUSSFile: string) {
+        this.log.trace("submitJob called with USS file %s", jobUSSFile);
+        return SubmitJobs.submitJobCommon(session, {jobUSSFile});
+    }
+
+    /**
+     * Submit a job that resides in a z/OS data set or USS file.
      * @static
      * @param {AbstractSession} session - z/OSMF connection info
      * @param {ISubmitJobParms} parms - parm object (see for details)
      * @returns {Promise<IJob>} - Promise that resolves to an IJob document with details about the submitted job
      * @memberof SubmitJobs
      */
-    public static async submitJobCommon(session: AbstractSession, parms: ISubmitJobParms) {
+    public static async submitJobCommon(session: AbstractSession, parms: ISubmitJobParms|ISubmitJobUSSParms) {
```

Comment:
⚠️ Is this a red flag ? ⚠️ 
`Function signature change`

Reply:
False alarm, `OR`ing interfaces for inputs should be fine. Not for outputs though 😋